temporian/core/operators/binary/relational.py

```diff
@@ -118,6 +119,10 @@ def equal(
     Returns:
         Node containing the result of the comparison.
     """
+
+    if not isinstance(input_2, Node):
+        return equal_scalar(input=input_1, value=input_2)
```

Comment:
This is good.
I'm assuming we don't need this for the rest of the ops because nobody will use them directly.

Reply:
exactly my thought.